CHANGES.txt

```diff
@@ -5,6 +5,12 @@ Changes for crate
 Unreleased
 ==========
 
+- BREAKING CHANGE: Improved SQLAlchemy's ``CrateDialect.get_pk_constraint`` to
+  be compatible with changes in CrateDB >=5.0.2. Since CrateDB 3.0.0 already,
```

Comment:
I think it should be _...compatible with changes in CrateDB>=5.1_. I updated PR title and all comments accordingly - could you reword commit message as well?